src/search/pdbs/cegar.cc

```diff
@@ -226,9 +227,10 @@ bool CEGAR::time_limit_reached(
 unique_ptr<PatternInfo> CEGAR::compute_pattern_info(Pattern &&pattern) const {
     vector<int> op_cost;
     bool compute_plan = true;
-    shared_ptr<PatternDatabase> pdb =
-        make_shared<PatternDatabase>(task_proxy, pattern, op_cost, compute_plan, rng, use_wildcard_plans);
-    vector<vector<OperatorID>> plan = pdb->extract_wildcard_plan();
+    PatternDatabaseFactory pdb_factory(
```

Comment:
As mentioned in our quick discussion, it's not so nice that this usage differs from the others.

Reply:
I fixed this by having two functions to compute a PDB: one with plan and one without.